src/dbt_osmosis/core/osmosis.py

```diff
@@ -410,6 +418,9 @@ def bootstrap_sources(self) -> None:
                                 "description": getattr(
                                     exp, "description", getattr(c, "description", "")
                                 ),
+                                "data_type": getattr(
+                                    exp, "dtype", getattr(c, "dtype", "")
```

Comment:
Should `getattr(c, "dtype", "")` be `getattr(c, "dtype")`?
Do you want to throw an error here so you can catch it and inform the user?
Or better yet not populate the key if there is not value? Does it matter?

Reply:
If dtype isn't available (model isn't materialised) it wouldn't update the descriptor, tested right now